config/envs/default.py

```diff
@@ -93,3 +93,5 @@ class DefaultConfig:
 
     SQLALCHEMY_DATABASE_URI = environ.get("DATABASE_URL")
     SQLALCHEMY_TRACK_MODIFICATIONS = False
+
```

Comment:
Hardcording URL's is not a good practice. Instead, can we configure urls from app bindings or env vars ?

Reply:
Yes - this is what I was going to do, but need this tracker done quickly as it's due to be used soon

Reply:
Removed the links here: https://github.com/communitiesuk/funding-service-design-application-store/pull/237/commits/53815f7f7ddfce093a7e4b9c8e41e1915cf538e3